src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2629,13 +2623,13 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirDBTemp =
                 PsyTdbFnHW(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy,
                            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat);
-            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate;
+            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate / PartLoadRatio;
```

Comment:
These are switched. Divide by PLR if cycling fan, LoadSideMassFlowRate is already full flow for constant fan.

Reply:
At line 2680 is:

    state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate = state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot;

So you really don't even need lines 2626, 2632 and 2680 since you are reading the node flow rate into AirMassFlowRate at line 1148 and haven't changed it anywhere.

Reply:
@rraustad I am a bit confused. You're right, the PLR is compressor PLR then the constant fan case shouldn't be divided by PLR (also applied to line 2608, will push a commit soon to address it). So if it's cycling fan, should we divide it? I think the `AirMassFlowRate` in this data struct is used as a timestep averaged value (they're also tracked by reporting variables), so PLR is supposed to be applied, right? 
I cannot agree more that those lines can be redundant, I think the whole LoadSideMassFlowRate var is useless with its equivalence to AirMassFlowRate.

Reply:
Correct for constant fan. If fan mode is cycling fan then the node AirMassFlowRate will be proportional to PLR, or the average over the time step, set by the parent. So you must divide by PLR. What you are actually doing is AirMassFlowRate/PLR = full air flow rate. So in both cases you are modeling the coil using the full air flow rate and then adjusting outlet conditions based on PLR and fan mode down in the CyclingScheme block at line 2615.